src/common/services/request/query/fetch-plays.js

```diff
@@ -46,12 +46,18 @@ export function FetchPlaysBySlugAndUser(playslug, username) {
   return payload;
 }
 
-export function FetchPlaysByFilter(where_clause) {
+export function FetchPlaysByFilter(where_clause, sortBy) {
   const payload = { ...BasiFetchParam };
   if (where_clause) {
     payload.where = where_clause;
   }
 
+  if (sortBy === 'Newest') {
+    payload.orderBy.created_at = 'desc';
+  } else {
+    payload.orderBy.created_at = 'asc';
+  }
+
```

Comment:
this too should be done on frontend since other two sorting being done on FE side.

Reply:
Ideally, we should be doing all sorting/filtering on the middleware/backend due to keep the frontend lightweight.

In this case, I was unable to query the data by _Most Liked_ and _Random_. hence I did it on the frontend side. I would love some help in modifying the query to do these on backend as well.

Reply:
I was able to sort by Most Liked using the query.
For Sort by `Random` - I am querying the plays based on descending order of description length and then sorting them randomly on frontend. 

I think querying based on random is not possible using graphql. Please correct me if I am wrong.

Reply:
Its good now then. It should be either on FE or from BE can't be half on FE & some on BE.